code/modules/research/designs.dm

```diff
@@ -298,21 +298,14 @@
 /datum/design/item/mining/AssembleDesignName()
 	..()
 	name = "Mining equipment design ([item_name])"
-/*
+
 /datum/design/item/mining/jackhammer
-	id = "jackhammer"
+	id = "CEC-2000 Laser Pickaxe"
```

Comment:
I'm not sure where this name came from, the item is called a rock saw
https://deadspace.fandom.com/wiki/Rock_Saw

Reply:
Will edit. Will edit the tool name itself while I'm at it.